### DISCUSSION TITLE
java.sql.SQLFeatureNotSupportedException: Unsupported feature: releaseSavepoint

### DISCUSSION THREAD

**Discussion by 1450793561** [apache/skywalking#11800] [Q&A]
Application success rate drop, through trace found the following error message, but from the backend application we didn't find "Java. SQL. SQLFeatureNotSupportedException: Unsupported feature: releaseSavepoint" similar exception

<img width="1162" alt="企业微信截图_4957125b-1bd6-439c-9065-a9e2fb8af7df" src="https://github.com/apache/skywalking/assets/31468018/8d37d0f4-6272-4c52-9bb8-56e266aa7633">

**Comment by 1450793561** [comment]
how do happen？

**Comment by wu-sheng** [comment]
The stack is clear, we are only manipulating your stack. You are seeing SkyWalking in the stack just because, this method(`releaseSavepoint`) is being monitored. But we don't trigger that.

<img width="921" alt="image" src="https://github.com/apache/skywalking/assets/5441976/41b6211d-c8c3-4d2c-9d84-d1c43d85dd25">

**Reply by wu-sheng** [reply]
You can see, Spring calls commits first, then it runs into oracle driver, we(SkyWalking) are just intercepting Oracle driver's methods.
